src/jit/lsraxarch.cpp

```diff
@@ -2824,6 +2824,10 @@ int LinearScan::BuildIndir(GenTreeIndir* indirTree)
         }
     }
 #ifdef FEATURE_SIMD
+    if (varTypeIsSIMD(indirTree))
+    {
+        SetContainsAVXFlags(true, genTypeSize(indirTree->TypeGet()));
```

Comment:
I wonder if there is an assert we could do in the emitter to catch these whenever we emit a relevant instruction....